assets/app/styles/_core.less

```diff
@@ -301,6 +314,11 @@ html, body {
   }
 }
 
+.required:before {
+  content: '* ';
```

Comment:
The space matters. Without it:

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/9044109/5d82fc56-39e8-11e5-9c79-1a0e58269cf0.png)